fetch.js

```diff
@@ -426,7 +435,11 @@
         xhr.setRequestHeader(name, value)
       })
 
-      xhr.send(typeof request._bodyInit === 'undefined' ? null : request._bodyInit)
+      if (request._bodyArrayBuffer) {
+        xhr.send(request._bodyArrayBuffer)
```

Comment:
Also, it would be good to make a test for this that performs a real HTTP request (in supporting browsers).